utils/getGithubId.js

```diff
@@ -0,0 +1,14 @@
+const logger = require('./logger')
+const userQuery = require('../models/users')
+
+const getGithubId = async (id) => {
+  try {
+    const details = await userQuery.fetchUser(id)
+    return (details.user.github_id)
+  } catch (err) {
+    logger.error(`Error while fetching user: ${err}`)
+    throw err
+  }
+}
+
```

Comment:
@swarajpure @whyDontI Maybe that function should be moved into `utils` for better visibility for reusability

Reply:
@ankushdharkar there are more functions related to users in userController, do we need to move all of them in that case?

Reply:
This is not required. The user data is already fetched and can be accessed with `req.userdata`.
Refer: https://github.com/Real-Dev-Squad/website-backend/blob/develop/middlewares/authenticate.js#L38

Reply:
UserController is separate because all the user related functions are there. We should import functions from there to where ever we need to use it.
I don't think we should to create a separate function just to access one key (github_id).

Reply:
@ankurnarkhede That is only true for logged in user, here we need user details with ID